test/misc.js

```diff
@@ -3,6 +3,28 @@ var cosmic = require('./fixtures/cosmic')
 var validator = require('../')
 var validatorRequire = require('../require')
 
+tape('invalid schemas', function(t) {
+  t.throws(function() {
+    validater(Buffer.from('foo'))
```

Comment:
It throws because `validater` is undefined due to a mistype (here and in all other checks below).
This test isn't testing anything.